core/src/main/java/bio/terra/pearl/core/dao/workflow/ParticipantTaskDao.java

```diff
@@ -100,12 +100,12 @@ public Optional<ParticipantTask> findTaskForActivityWithCompletionTime(UUID ppUs
                                 and study_environment_id = :studyEnvironmentId
                                 and %s
                                 limit 1
-                                """.formatted(tableName, approximateDateMatchQueryStr("completed_at", ":completedAt::timestamp", "43200"))
+                                """.formatted(tableName, approximateDateMatchQueryStr("created_at", ":createdAt::timestamp", "60"))
                         )
                         .bind("ppUserId", ppUserId)
                         .bind("activityStableId", activityStableId)
                         .bind("studyEnvironmentId", studyEnvironmentId)
-                        .bind("completedAt", Timestamp.from(completedAt))
+                        .bind("createdAt", Timestamp.from(createdAt))
```

Comment:
I cannot remember why I was matching tasks on complete time and not creation time. It makes it more robust to incomplete surveys this way - I can't seem to remember why I thought that restriction was good (if I did)